pom.xml

```diff
@@ -563,6 +563,23 @@
             <groupId>org.janelia.saalfeldlab</groupId>
             <artifactId>n5-imglib2</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Please have a look at the CI logs -- the new dependencies seem to include more slf4j bindings, leading to duplicate classes. They can be excluded from a specific dependency using the procedure described at https://maven.apache.org/guides/introduction/introduction-to-optional-and-excludes-dependencies.html#dependency-exclusions